lab-cameron/lib/fs-readfile.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const fs = require('fs');
+
+// I LEFT IN THESE CONSOLE LOGS TO GIVE MY TESTS MORE CONTEXT AND AS PROOF OF ASYNCHRONICITY //
```

Comment:
**TODO FOR LOST POINTS**

I like that you're explaining why, but we're still getting tough this week now on leaving console.logs in. Try to manage your console.logs and comments on another branch and submit a clean branch to Canvas for grading.